tasks.py

```diff
@@ -281,6 +281,14 @@ def help(c):
     except:
         pass
 
+@task
+def generate_certificates(c):
+    """
+    Generates the cert.pem and cert-key.pem files
+    """
+    with c.cd(c.project_directory + 'infrastructure/docker/services/router/etc/ssl/certs/'):
```

Comment:
I did not test it, but I think it does not work. It misses a leading `/` before infrastructure.
I say that, because It's a best practice to never put trailing `/` in a variable container a path